pkg/reportingplugins/mercury/mercury.go

```diff
@@ -360,6 +362,12 @@ func (rp *reportingPlugin) Observation(ctx context.Context, repts ocrtypes.Repor
 		p.CurrentBlockHashValid = true
 	}
 
+	if obs.CurrentBlockTimestamp.Err != nil {
+		obsErrors = append(obsErrors, pkgerrors.Wrap(obs.CurrentBlockTimestamp.Err, "failed to observe CurrentBlockTimestamp"))
+	} else {
+		p.CurrentBlockTimestamp = obs.CurrentBlockTimestamp.Val
```

Comment:
do we need `p.CurrentBlockTimestamapValid` field here similar to other block values?

Reply:
Yes it can be missing